fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -107,7 +107,7 @@ private void handleInitDb() {
             String[] parts = identifier.trim().split("\\s+");
             if (parts.length == 2) {
                 if (parts[0].equalsIgnoreCase("catalog")) {
-                    ctx.getGlobalStateMgr().changeCatalog(ctx, parts[1]);
+                    ctx.getGlobalStateMgr().changeCatalogDb(ctx, parts[1] + ".");
```

Comment:
Although the code can be reused, but the readability is weaker. suggest another way

Reply:
Have restored the `changeCatalog` function.